sockshop/payment/service_unit_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 )
 
-func TestAuthorise(t *testing.T) {
+func TestUnitAuthorise(t *testing.T) {
```

Comment:
It'd be a bit weird if all your unit tests had a Unit prefix. IMO better to leave these in the normal form, and select for only unit testing, if you want to do that, by e.g. `go test -test.run="Test(?!Component).*"`. But I guess it's minor.